rfcs/0000_off_object_observations.md

```diff
@@ -0,0 +1,237 @@
+- Start Date: 2025-08-12
+- RFC PR: (leave this empty, it will be filled in after RFC is merged)
+
+# Sending and Processing Off-Object Observations in Monty
+
+**Note**: This RFC is a spawn from [Question 3 of the Intelligent Resampling in Monty PR (closed)](https://github.com/thousandbrainsproject/tbp.monty/pull/366). For richer theoretical discussions please check the previous RFC. This RFC will put more emphasis on implementation. 
+
+## Question and Goal
+
+The question that we are addressing is: How can we send and process off-object observations in Monty?
+
+The goal of the RFC is to layout current state of Monty and implementation plan to process off-object observations. 
+
+## Summary
+
+This RFC proposes extending Monty's sensory processing pipeline to handle off-object observations (when a sensor has moved beyond an object's reference frame). Currently, these observations are filtered out and not passed to Learning Modules (LMs), limiting our ability to leverage prediction errors for penalizing hypotheses' evidence. By defining "null features" to represent the absence of morphological features, modifying the `FeatureChangeSM` to forward these observations, and updating the Learning Module to process off-object observations, we can enable more intelligent hypothesis management when sensors transition from on-object to off-object status. This enhancement will improve Monty's ability to recognize when it has moved off an object and reduce evidence for hypotheses that predict the sensor should still be detecting object features. 
+
+<img src="0000_off_object_observations/off_object_observation.png" alt="Off-object observation" />
```

Comment:
Is there a reason we're using an HTML literal for this? [Github Markdown supports image markup](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images).

```suggestion
![Off-object observation](0000_off_object_observations/off_object_observation.png)
```